### PR TITLE
Updating github link for EmojiReactions

### PR DESCRIPTION
**Pull request by ginotoro**
Updating github link for EmojiReactions from MessageReactions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This repository contains all of the code necessary to run and test the Ably exam
 Click on any of the links below to find out more specific information on each example:
 
 - [Avatar Stack](/src/components/AvatarStack/README.md)
-- [Message Interactions](/src/components/MessageReactions/README.md)
+- [Message Interactions](/src/components/EmojiReactions/README.md)
```
Comment (mohyour) [inline README.md:8]: Message Reactions
```suggestion
- [Message Reactions](/src/components/EmojiReactions/README.md)
```

Comment (mohyour) [reply README.md:8]: That should be Emoji Reactions
```suggestion
- [Emoji Reactions](/src/components/EmojiReactions/README.md)
```

Sorry, that should be emoji reactions

Comment (mohyour) [reply README.md]: @ginotoro I updated this to be `Emoji Reactions`

Comment (ginotoro) [reply README.md]: Great. Can you review and merge the changes?

**Review by mohyour — APPROVED** [review]

**Pull request merged**